src/index.js

```diff
@@ -137,9 +137,9 @@ module.exports = class Bitwap {
       cb(err, block)
     }
 
-    this.getBlocks([key], (err, res) => {
-      if (err) {
-        return done(err)
+    this.getBlocks([key], (errs, res) => {
+      if (errs) {
+        return done(errs[0])
```

Comment:
@diasdavid done, please give this another round of review so I can merge it soon :)